ui/actions/nuxeo-favorites-toggle-button.js

```diff
@@ -147,25 +147,29 @@ import '../nuxeo-button-styles.js';
     _toggle() {
       if (!this.favorite) {
         this.$.opAdd.execute().then(() => {
-          this.dispatchEvent(
-            new CustomEvent('added-to-favorites', {
-              composed: true,
-              bubbles: true,
-              detail: { doc: this.document },
-            }),
-          );
-          this._setFavorite(true);
+          if (window.confirm(this.i18n('favoritesToggleButton.confirm.add')) === true) {
```

Comment:
Please note that we are asking the user to confirm AFTER we already executed the operation, which is not what we want here.